app/views/pages/medicaid/amounts_loan_interest.erb

```diff
@@ -0,0 +1,25 @@
+<div class="step-header">
+  <a href="/pages/medicaid/amounts_other_income" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Income & Expense Amounts
```

Comment:
Small little nitpick, but - I'd suggest changing all of the lone ampersands here to use their html encoded counterpart `&amp;`

(discussion here: https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp)